### DISCUSSION TITLE
2024-08-23 - SEO Tools - SEMrush - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9046] [Chewing the Fat]
https://www.ssw.com.au/rules/seo-tools

What SEO tools do you use on your site?
Any tips to be the 1st result on Google?

**Comment by Freego1783** [comment]
I've never been too big on SEO, never really worked on it. But SEMRush looks promising I'll consider it if I need a good SEO tool!

**Comment by PothieuG** [comment]
I don't really have SEO experience. I wasn't in charge of it for my professional project, and all my side projects are for personal use. 
However, helping my sister deploy a WordPress site showed me that WordPress has really good built-in SEO extensions.

**Comment by tiagov8** [comment]
Google Analytics and Search Console are must haves for any website.

I don't have a lot of experience with SEMRush but I like what 've seen so far.

For all internal projects we also use [CodeAuditor](https://codeauditor.com/) - A clean code is also important to good SEO

**Comment by drwharris** [comment]
SEO is one of those things that makes me go
![SighDoIHaveToGIF](https://github.com/user-attachments/assets/b5ada414-201c-4189-bc04-611d1238c9f3)

**Comment by brookjeynes-ssw** [comment]
Never worked much with SEO but Semrush seems like a pretty comprehensive tool to use when you need in-depth insights.

**Comment by steven0x51** [comment]
I don’t have much experience on SEO tools. It’s interesting topic, I’m ready to explore and start to think how to improve the visibility of a web site

**Comment by Levijj22** [comment]
I wonder if/how AI has affected SEO. 
I imagine there will be tools to help with this soon.

**Comment by bettybondoc** [comment]
Even back when I worked in Marketing, SEO has always been an enigma.

Plus, the rules changed a lot over the years. Keyword stuffing used to be cool until it wasn't.

**Comment by RobJT-wq** [comment]
I've never thought about SEO tools before but it makes sense to use them to improve SEO rankings

**Comment by StefStar** [comment]
In my previous roles I used semrush in a basic but crucial way, mainly for keyword research for ads. I did not realise that it can be crucial for other optimisations in this space.

**Comment by jakebayliss** [comment]
Never touched SEO but definitely an important part of any project :)

**Comment by edkranz** [comment]
Did not know there were complete tool suites for SEO. Good to know. SEMRush seems like it takes the guesswork out of SEO

**Comment by Marxoz** [comment]
I haven't used any SEO tools but getting into it now. They are so crucial to any product / service page that you will always need someone with knowledge of SEO and the tools on your team

**Comment by ricksu978** [comment]
I use Google Structure data test tool
https://developers.google.com/search/docs/appearance/structured-data

> Structured data refers to a standardized format for providing information about a page and classifying the content on the page. It helps search engines understand the context of the content, which can lead to enhanced display features in search results, such as rich snippets.

**Comment by Calinator444** [comment]
> What SEO tools do you use on your site?

We use CodeAuditor, SEMrush Google Analytics, and Azure Application Insights

> Any tips to be the 1st result on Google?

I can't assume sole responsibility for our Google Ranking because I'd mostly attribute that to the work the seniors working on our site have put in.

For a few tips though:
- Make sure all of your web pages have meta descriptions
- Ensure there are a lot of back links going to your website
- Make sure there are no broken links on your site
- Avoid cumulative layout shifts (use skeleton loading for content that loads client side)
- If your website fetches from a data source consider adding pagination to slice content and reduce the size of your initial page
- Make sure your pages don't take too long to load, use Static Site Generation to reduce external API calls & reduce the size of your images where applicable

**Comment by vladislav-kir** [comment]
I have never worked on SEO, but it's great to learn more about it. SEMrush looks very promising, although quite pricy

**Comment by danielmackay** [comment]
> What SEO tools do you use on your site?

On my blog ([dandoescode.com](https://www.dandoescode.com)), I use a combination of Vercel Analytics and Google Analytics.  I find it interesting to see how people are arriving on my site and what is popular, but I don't manually tweak the SEO metadata.

> Any tips to be the 1st result on Google?

Give Marketing a call. 😃

**Comment by JackDevAU** [comment]
Keen to learn more but I haven't explored much into the SEO space. SEMRush seems interesting!

**Comment by jeoffreyfischer** [comment]
On Website, we use Semrush, PageSpeed and Azure Failed requests - we are planning to add Google Console to compare the results and ensure the numbers are similar

**Comment by joshbermanssw** [comment]
Slowly learning more about SEO with Google Analytics on tina.io

**Comment by matt-goldman-ssw** [comment]
>What SEO tools do you use on your site?

None really

> Any tips to be the 1st result on Google?

Pay up. Google have been exposed for promoting paid search results. You're only getting to the top if you grease the right palms.

**Reply by Calinator444** [reply]
> Pay up. Google have been exposed for promoting paid search results. You're only getting to the top if you grease the right palms.

Is this excluding Google ads? Because we put a fair amount of effort into tailoring our keywords on the website and following best practices for content to improve SEO. I don't want to waste time doing that if it has little to no impact on our rankings.

**Comment by jaydenalchin** [comment]
SEO is top of my list of things I'd love to learn more about! I have only approximate knowledge of some tools and practices.

**Comment by TobyChurches** [comment]
Im not really familiar with these tools - We should improve this for the TimePro marketing page

**Comment by chrisschultzssw** [comment]
> What SEO tools do you use on your site?

N/A - I don't have a site 

> Any tips to be the 1st result on Google?

💵💵💵

**Comment by awaltos** [comment]
I have used SEMrush a little bit and not only does it seem packed with features, it is also super easy to use - great UX. But it is not too cheap, especially if you need multiple user accounts. Tips on being first are to try imagine what your customer is typing in to find your product and build on those keywords. But this is not enough, you also need backlinks, and good content, authority and reliability ranking, etc, etc.

**Comment by amankumarrr** [comment]
I use SEMrush for the SSW Website, primarily to identify potential issues, broken links, and toxic backlinks, all of which significantly contribute to improving our SEO. While I haven’t yet explored the marketing, keyword analysis, and other non-technical aspects, I recognize that SEMrush offers a comprehensive suite of tools beyond technical SEO that we could leverage to enhance our search engine rankings further

**Comment by christoment** [comment]
Google analytics and Google search console is good to start with 🙂

**Comment by leahy268** [comment]
I've only looked over Tanya's shoulder while she is using SEMrush but it looks like a great tool.

**Comment by Tanya-Leahy** [comment]
I'm using SEMrush at the moment for improving our consulting pages and I am finding the tool really helpful.  Must have tool!

**Comment by Dhruv-0987** [comment]
What SEO tools do you use on your site?
Haven't had any experience with it but Semrush is a good place to start

**Comment by bahjatalaadelssw** [comment]
SEO has become more plain language and simpler over the years.

**Comment by tkapa** [comment]
I've previously used Google Analytics / Vercel Analytics/Metrics.

I'm far from an SEO master, and don't find it particularly interesting to work with, so I don't think I'm the best person for suggestions 🤣

**Comment by jimmidier** [comment]
> What SEO tools do you use on your site?

None

> Any tips to be the 1st result on Google?

None. I'd love to see wise suggestions on this topic

**Comment by jerryluossw** [comment]
Most of the projects I work on are internal systems, so there is little need for SEO, I don’t have much experience on SEO tools.

**Comment by Aibono1225** [comment]
> What SEO tools do you use on your site?

Google Analytics

> Any tips to be the 1st result on Google?

None

**Comment by tino-liu** [comment]
I have never really worked with SEO before. But i do understand that it is very important for business exposure.
There is this joke that I like:
```
"Where do you hide a body?"
"In Google's page 2"
```
So we gotta aim for the top in page 1!

**Comment by babakamyljanovssw** [comment]
> What SEO tools do you use on your site?

No, I haven't used it yet

> Any tips to be the 1st result on Google?

None. First I would like to learn more about these tools.

**Comment by ozair-ashfaque** [comment]
No - I never used SEO tools before

**Comment by KristenHu** [comment]
I never used SEO tools before, but it looks pretty interesting🤔

**Comment by miichaelsmedley** [comment]
> https://www.ssw.com.au/rules/seo-tools
> 
> What SEO tools do you use on your site? Any tips to be the 1st result on Google?

SEM Rush and hREF are tools I use.  Google analytics also helps.

Very important in SEO to understand your viewer's search behaviour 1st and use the tools second.

**Comment by stephanfakossw** [comment]
Never used SEO - it's not one of those tools that accountants use 😊

**Comment by jimmy1992abc** [comment]
This is the first time I know about SEO (I knew similar things like the backlog of Google Ads)
I am not a potential user of these SEOs, this CTF is just amazing in that I learnt a lot about how to adjust keywords in a website to give it more visibility.

**Comment by MarcusIrmscherSSW** [comment]
Good SEO is vital to boost traffic, and can make a huge difference.

**Comment by isaaclombardssw** [comment]
> What SEO tools do you use on your site?

I've used the free tools (google analytics, etc.) before.

> Any tips to be the 1st result on Google?

Would love to know. Intuitively, aiming for low-competition or longer keywords seems to help.

**Comment by duanxinhuan** [comment]
YakShaver is not on the first page, I'm sad

**Reply by Calinator444** [reply]
I have no idea what this is but I think we should sue 😆

<img width="857" alt="image" src="https://github.com/user-attachments/assets/b1ea8d73-9506-406f-b427-b84b18e5e6a3">

**Figure**: **Yakshaver first result on Google**

**Comment by MattParkerDev** [comment]
> What SEO tools do you use on your site?

I have used Google Analytics/Search console before.

> Any tips to be the 1st result on Google?

don't make your brand name/website too common

**Comment by PennyWalker** [comment]
Google Analytics and Search Console are important ones to be using, but SEMRush is a great tool for research as well as optimisation. I think we should be paying for 1 license that has even more features like the web page creator that helps you design great content for SEO. :)

**Comment by calumjs** [comment]
SEO seems back to front to me - if you pack your site full of pages that contain certain keywords just to improve their search ranking, I don't think your site will be interesting to a human.
I'd rather just organically produce the content and rely on other advertising methods to get the word out.
I hope AI assisted page indexing allows this to become a thing of the past.

**Comment by Marxoz** [comment]
> What SEO tools do you use on your site?

I have only used Google Analytics/Search console before.

> Any tips to be the 1st result on Google?

The content should be relevant and up to date
